library/include/ck/library/utility/host_tensor.hpp

```diff
@@ -271,6 +271,7 @@ struct Tensor
     ~Tensor() = default;
 
     Tensor& operator=(const Tensor&) = default;
+
```

Comment:
Let's not make this file unnecessarily "wider" :)

Reply:
Oops, fixed in [a7da01f](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/375/commits/a7da01f0c12537fbf5df4c8d0b3cd8ee1b7f655e)